apps/browser/qml/pages/components/PermissionCreateDialog.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.WebView.Controls 1.0
+
+Dialog {
+    property alias label: valueButton.label
+    property int capability: indexToCapability(valueButton.currentIndex)
+    property alias uri: textField.text
+    readonly property var _regExp: new RegExp("^(http|https):\/\/")
+
+    function indexToCapability(index) {
+        if (index === 0) { // index for allow context menu
+            return PermissionManager.Allow
+        } else {
+            return PermissionManager.Deny
+        }
+    }
+
+    canAccept: (textField.text !== "" && !textField.errorHighlight)
+
+    DialogHeader {
+        id: header
+
+        //: Accept text when add new permission
+        //% "Add"
+        acceptText: qsTrId("sailfish_browser-he-add")
+    }
+
+    Column {
+        width: parent.width
+        anchors.top: header.bottom
+
+        TextField {
+            id: textField
+
+            width: parent.width
+            //% "Website addres"
+            placeholderText: qsTrId("sailfish_browser-he-website-address")
+            label: {
+                if (errorHighlight) {
+                    //% "Should begin with http:// or https://"
```

Comment:
I'd suggest changing this to "Must begin with" (since "should" implies it's optional).